deployments/fargate/README.md

```diff
@@ -0,0 +1,143 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS task in a **sidecar** container. We are using the term
+"sidecar" loosely here to mean having the Collector container as a sidecar to multiple or
+all monitored application containers in the task. We do not mean a Collector sidecar container
+per monitored application container as in the strict sense.
+
+Requires Connector release v0.30.0 or newer which corresponds to image tag 0.30.0 and newer.
+See image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+## Getting Started
```

Comment:
Immediately following this header have a code block with something a user can copy and paste to get the connector working on Fargate.

Reply:
Added default container definition JSON for Collector.